### PR TITLE
[Display Settings]

### PR DESCRIPTION
**Pull request by GhostofGeeeee**
BoxInfo.setItem("SuspendPowerLed", fileExists("/proc/stb/power/suspendled") **and MODEL not in ("multibox", "multiboxse"))**

### CODE REVIEW

File: lib/python/Components/SystemInfo.py [hunk 1]
```diff
@@ -457,7 +457,7 @@ def getWakeOnLANType(fileName):
 BoxInfo.setItem("SoftCam", hasSoftcam())
 BoxInfo.setItem("StandbyPowerLed", fileExists("/proc/stb/power/standbyled"))
 BoxInfo.setItem("STi", SOC_FAMILY.startswith("sti"))
-BoxInfo.setItem("SuspendPowerLed", fileExists("/proc/stb/power/suspendled"))
+BoxInfo.setItem("SuspendPowerLed", fileExists("/proc/stb/power/suspendled") and MACHINEBUILD not in ("maxytecmulti", "maxytecmultise", "anadolmulti", "anadolmultitwin", "anadolmultiboxse", "axmulticombo", "axmultitwin", "axmultiboxse", "novaler4k", "novaler4kse", "novaler4kpro"))
```
Comment (jbleyel) [inline lib/python/Components/SystemInfo.py:460]: Where do you get this list of models?

File: data/setup.xml [hunk 2]
```diff
@@ -151,8 +151,8 @@
 			<item level="1" text="Show PiP in the Front Display" description="You can Display PiP with or without OSD Menu.">config.lcd.modepip</item>
 			<item level="1" text="Set Framerate for MiniTV" description="Lower Framerate saves CPU Time.">config.lcd.fpsminitv</item>
 		</if>
-		<item level="1" text="REC Symbol in use" description="Enable blinking rec symbol on the LCD Display.">config.usage.blinking_rec_symbol_during_recording</item>
-		<item level="1" text="REC Symbol in Standby" description="Enable blinking rec symbol on the LCD Display in Standby.">config.usage.blinking_display_clock_during_recording</item>
+		<item level="1" text="REC Symbol in use" description="Enable blinking rec symbol on the LCD Display." requires="LCDSKINSetup">config.usage.blinking_rec_symbol_during_recording</item>
```
Comment (jbleyel) [inline data/setup.xml:154]: Are you sure about the "requires"?

File: lib/python/Components/SystemInfo.py (hunk 1 above)
Comment (GhostofGeeeee) [reply lib/python/Components/SystemInfo.py:460]: https://github.com/oe-alliance/build-enviroment/blob/5.3/Makefile

File: data/setup.xml (hunk 2 above)
Comment (GhostofGeeeee) [reply data/setup.xml:154]: I think blinking is set in a display skin and these are related to
BoxInfo.setItem("LCDSKINSetup", fileExists("/usr/share/enigma2/display")).

File: lib/python/Components/SystemInfo.py (hunk 1 above)
Comment (jbleyel) [reply lib/python/Components/SystemInfo.py:460]: Where do you find not SuspendPowerLed support in this makefile?

File: data/setup.xml (hunk 2 above)
Comment (GhostofGeeeee) [reply data/setup.xml:154]: No `fileExists("/usr/share/enigma2/display")` No blinking

File: lib/python/Components/SystemInfo.py (hunk 1 above)
Comment (GhostofGeeeee) [reply lib/python/Components/SystemInfo.py:460]: You said its a driver bug, i think not only one of these box type have this.

Comment (jbleyel) [reply lib/python/Components/SystemInfo.py:460]: Then we need to find first all affected models and not only guessing.

Comment (GhostofGeeeee) [reply lib/python/Components/SystemInfo.py:460]: I need to remove "novaler4kpro".
All others are:
MACHINE=multibox
MACHINE=multiboxse

**Comment by jbleyel** [comment]
This should be done in c code and not here.

**Comment by GhostofGeeeee** [comment]
OK, but i don't know how!
This looked good to me.

**Comment by jbleyel** [comment]
What exactly is not working here -> "Power LED State in Deep-Standby"
What exact box model do you have with this problem?

**Comment by GhostofGeeeee** [comment]
> What exactly is not working here -> "Power LED State in Deep-Standby"

Power LED State in Deep-Standby is allways ON(Off doesn't work)
> What exact box model do you have with this problem?

multibox

**Pull request closed without merging**